webxr-api/device.rs

```diff
@@ -57,4 +58,13 @@ pub trait Device: 'static {
     /// Inputs registered with the device on initialization. More may be added, which
     /// should be communicated through a yet-undecided event mechanism
     fn initial_inputs(&self) -> Vec<InputSource>;
+
+    /// Sets the event handling callback
+    fn set_event_callback(&mut self, callback: Box<dyn EventCallback>);
+
+    /// Whether the device is still connected
+    fn connected(&mut self) -> bool;
+
+    /// Quit the session
+    fn quit(&mut self);
```

Comment:
Can we just make this the `Drop` code for the device?

Reply:
Then XRSession would have to store an `Option<Session>` for no good reason.